lorien/InfiniteCanvas/Tools/RectangleTool.gd

```diff
@@ -36,6 +36,10 @@ func _make_rectangle(pressure: float) -> void:
 	pressure = pressure_curve.sample(pressure)
 	
 	var bottom_right_point := _cursor.global_position
+	# draw a square if Shift is pressed
+	if Input.is_key_pressed(KEY_SHIFT):
+		var drag_dir = _cursor.global_position - _start_position_top_left
```

Comment:
Please use `:=` here for drag_dir, so that it's properly typed

Reply:
Yep, I fixed it
I may review the other PR for the color picker: I haven't been so precise in the other one